app.js

```diff
@@ -41,49 +40,35 @@ app.configure(function() {
   });
 });
 
-app.post('/add-session', limiter, function (req, res) {
```

Comment:
`app.post('/add-session',...` is old stuff used for 2016 proposal flow. We don't need it anymore.